src/externs.ts

```diff
@@ -78,24 +79,88 @@ export function getGeneratedExterns(externs: {[fileName: string]: string}): stri
   return allExterns;
 }
 
+/**
+ * Returns a mangled version of the module name (resolved file name) for source file.
+ *
+ * The mangled name is safe to use as a JavaScript identifier. It is used as a globally unique
+ * prefix to scope symbols in externs file (see code below).
+ *
+ * @param contextFileName if given is used as the context file path to resolve the source file's
+ *     name against.
+ */
+export function mangledModuleName(
```

Comment:
This name describes what the function does internally, but it might be better to give it a name that describes its purpose (e.g. `moduleNameAsValidIdentifier` or some such).  Pretty minor though.

Reply:
Done.